### PR TITLE
Document ROI and stddev features

### PR DESCRIPTION
**Pull request by twentyse7en**
Added description and screenshot of
ROI and stddev in usage docs.
ISSUE #591 
## Contributor Checklist:

* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -123,6 +123,11 @@ in scan coordinates on the right.
 
 ..  figure:: ./images/use/apply.png
 
+Instead of average, you can select "Standard Deviation". This calculates
+standard deviation of all detector frame.
```
Comment (sk1p) [inline docs/source/usage.rst]: Better: "all detector frame**s**". Maybe also some better wording can be found to describe over which axes the reduction is done, but maybe the screenshot is enough to intuitively understand this.

In my opinion, the screenshot should should have stddev already enabled and show the result, in addition of showing the drop-down. Similar to the rect screenshot already showing the rectangular ROI.

File: docs/source/usage.rst [hunk 2]
```diff
@@ -136,6 +141,13 @@ below the frame window on the left.
 
 ..  figure:: ./images/use/pick_frame.png
 
+If you are interested in limited region, ROI provide option to select rectangle,
```
Comment (sk1p) [inline docs/source/usage.rst]: Suggestion: "If you are interested in **a** limited region, **the** ROI **dropdown** provide**s**  **the** option to select **a** rectang**ular** ~~disk~~ region

**Comment by twentyse7en** [comment]
Added changes as requested.

File: docs/source/changelog/documentation/doc_roi_std.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[Documentation] Added ROI and stddev features
+=============================================
+
+ * Added screenshots and description of ROI
```
Comment (sk1p) [inline docs/source/changelog/documentation/doc_roi_std.rst]: Sorry I didn't notice this in the previous review round: could you also add a reference to the pull request number here? Then it should be ready for merging. Thanks!

Comment (twentyse7en) [reply docs/source/changelog/documentation/doc_roi_std.rst]: No problem :+1:

**Comment by twentyse7en** [comment]
Hey @sk1p , could you please help me with  [travis CI failure](https://travis-ci.org/LiberTEM/LiberTEM/builds/660207595?utm_source=github_status&utm_medium=notification)

**Comment by sk1p** [comment]
Sorry, you are hitting https://github.com/LiberTEM/LiberTEM/issues/674 - this is not a problem in your changes. Will merge your PR now!

**Pull request merged**